inc/admin-interface.php

```diff
@@ -36,6 +35,160 @@ function bootstrap() {
+	add_filter( 'pantheon_cache_max_age_input', __NAMESPACE__ . '\\update_default_ttl_input' );
+	add_filter( 'pantheon_cache_max_age_input_allowed_html', __NAMESPACE__ . '\\max_age_input_allowed_html' );
+}
+
+/**
+ * Enqueue admin assets.
+ *
+ * @since 2.0.0
+ * @return void
+ */
+function enqueue_admin_assets() {
+	$screen = get_current_screen();
+
+	if ( 'settings_page_pantheon-cache' !== $screen->id ) {
+		return;
+	}
+
+	// If WP_DEBUG is true, append a timestamp to the end of the path so we get a fresh copy of the css.
+	$debug = defined( 'WP_DEBUG' ) && WP_DEBUG ? '-' . time() : '';
+	wp_enqueue_style( 'papc-admin', plugin_dir_url( __DIR__ ) . 'assets/css/styles.css', [], '2.0.0' . $debug );
+}
+
+/**
+ * Add a header to the max-age setting field.
+ *
+ * @since 2.0.0
+ * @return string
+ */
+function add_max_age_setting_header() {
+	ob_start();
+	?>
+	<div class="pantheon-cache-default-max-age">
+		<?php
+		// Bail early if the header is disabled.
+		if ( ! apply_filters( 'pantheon_apc_max_age_header_enabled', true ) ) {
+			return ob_get_clean();
+		}
+		?>
+		<span class="pantheon-cache-default-max-age-info-bar"><i class="dashicons dashicons-info"></i><?php echo wp_kses_post( 'Boost site speed with a higher GCDN cache max age.', 'pantheon-advanced-page-cache' ); ?></span>
+	<?php
+	return ob_get_clean();
+}
+
+/**
+ * Add a description to the max-age setting field.
+ *
+ * @since 2.0.0
+ * @return string
+ */
+function add_max_age_setting_description() {
+	$filtered = has_filter( 'pantheon_cache_default_max_age' );
+	$above_recommended_message = __( 'Your cache maximum age is currently <strong>above</strong> the recommended value.', 'pantheon-advanced-page-cache' );
+	$below_recommended_message = __( 'Your cache maximum age is currently <strong>below</strong> the recommended value.', 'pantheon-advanced-page-cache' );
+	$recommended_message = __( 'Your cache maximum age is currently set to the recommended value.', 'pantheon-advanced-page-cache' );
+	$recommendation_message = get_current_max_age() > WEEK_IN_SECONDS ? $above_recommended_message : ( get_current_max_age() < WEEK_IN_SECONDS ? $below_recommended_message : $recommended_message );
+	$filtered_message = $filtered ? sprintf(
+		// translators: %s is the humanized max-age.
+		__( 'This value has been hardcoded to %s via a filter.', 'pantheon-advanced-page-cache' ),
+		'<strong>' . humanized_max_age() . '</strong>'
+	) : '';
+	$pantheon_cache = get_option( 'pantheon-cache', [] );
+	$custom_ttl = isset( $pantheon_cache['default_ttl'] ) && ! array_key_exists( $pantheon_cache['default_ttl'], max_age_options() );
+	$filtered_message .= $custom_ttl && ! $filtered ? '<br />' . __( '<strong>Warning:</strong>The cache max age is not one of the recommended values. If this is not intentional, you should remove this custom value and save the settings, then select one of the options from the dropdown.', 'pantheon-advanced-page-cache' ) : '';
```

Comment:
This is dense to read and parse out reading code but not going to block on it. For example, if $custom_ttl is a bool, the variable name should read as one.

https://github.com/pantheon-systems/Standards/blob/b0a15d37e61461226f1d6061bb3ac5d087548948/Data_Standards.md?plain=1#L15-L16

I know you love ternary operators but it's a pain long-term.